config/src/main/kotlin/uk/govuk/app/config/data/remote/model/ReleaseFlags.kt

```diff
@@ -5,5 +5,6 @@ import com.google.gson.annotations.SerializedName
 data class ReleaseFlags(
     @SerializedName("onboarding") val onboarding: Boolean,
     @SerializedName("search") val search: Boolean,
+    @SerializedName("recentActivity") val recentActivity: Boolean,
```

Comment:
Kept naming as Recent Activity in app code to match remote config as was told this flag will have larger scope in future than just show/hide Pages You've Visited.